src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -274,6 +274,10 @@ Adds an integration to your configuration. Read more in [the integrations guide]
 
 Launches the Astro Docs website directly from the terminal.
 
+## `astro info`
+
+Reports useful information about the Astro environment. Useful to providing information when opening an issue.
```

Comment:
``````suggestion
Reports useful information about your current Astro environment. Useful for providing information when opening an issue.

```shell
astro info
```

Example output:

```
Astro version            v2.6.6
Package manager          pnpm
Platform                 darwin
Architecture             arm64
Adapter                  @astrojs/vercel/serverless
Integrations             None
```
``````
Snagged this from the changeset!